packages/protocol/test/tokenomics/blockFee.test.ts

```diff
@@ -0,0 +1,135 @@
+import { expect } from "chai";
+import { BigNumber, ethers } from "ethers";
+import { AddressManager, TaikoL1 } from "../../typechain";
+import { TestTkoToken } from "../../typechain/TestTkoToken";
+import Proposer from "../utils/proposer";
+
+import sleep from "../utils/sleep";
+import { deployTaikoL1 } from "../utils/taikoL1";
+import { initTokenomicsFixture, onNewL2Block } from "./utils";
+
+describe("tokenomics: blockFee", function () {
+    let taikoL1: TaikoL1;
+    let l2Provider: ethers.providers.JsonRpcProvider;
+    let proposerSigner: any;
+    let genesisHeight: number;
+    let genesisHash: string;
+    let tkoTokenL1: TestTkoToken;
+    let l1AddressManager: AddressManager;
+    let interval: any;
+
+    beforeEach(async () => {
+        ({
+            taikoL1,
+            l2Provider,
+            proposerSigner,
+            genesisHeight,
+            genesisHash,
+            tkoTokenL1,
+            l1AddressManager,
+            interval,
+        } = await initTokenomicsFixture());
+    });
+
+    afterEach(() => clearInterval(interval));
+
+    it("expects the blockFee to go be 0 when no periods have passed", async function () {
```

Comment:
Initially it is 0, i Think because no bootstrapHalvingPeriods have passed, and the function that calculates the blockFee with no periods passed returns 0 in that situation.